core/src/main/java/com/datastax/oss/driver/internal/core/adminrequest/AdminRow.java

```diff
@@ -50,10 +51,12 @@ public AdminRow(
     this.protocolVersion = protocolVersion;
   }
 
+  @Nullable
```

Comment:
Added nullability annotations here to better understand what's happening in `DefaultTopologyMonitor`. This is creating warnings in many other places though, I didn't fix them as we already have JAVA-2284.